config.md

```diff
@@ -27,6 +27,7 @@ Each container has exactly one *root filesystem*, specified in the *root* object
 
 * **`path`** (string, required) Specifies the path to the root filesystem for the container. A directory MUST exist at the path declared by the field.
 * **`readonly`** (bool, optional) If true then the root filesystem MUST be read-only inside the container. Defaults to false.
+If readonly is present and set to true, but not supported by the platform, the platform MAY ignore.  
```

Comment:
On Thu, Jun 09, 2016 at 10:25:55PM -0700, Aleksa Sarai wrote:
> For the Microsoft case, just make it so that your runtime errors out
> if a user specifies readonly (though we should make it clear
> somewhere in the spec whether or not all runtimes need to implement
> all optional options).

I've taken a stab at part of the general behavior in #559.  I'm not
quite sure how to flag options which a runtime can not support while
still being certified.  Some thoughts on that in #559 too.